lib/msal-browser/tsconfig.json

```diff
@@ -2,8 +2,8 @@
     "extends": "./tsconfig.base.json",
     "compilerOptions": {
         "moduleResolution": "node",
-        "module": "es6",
-        "target": "es5",
+        "module": "ESNext",
```

Comment:
We have a separate item to bump module/target for all modules. Would you mind reverting these changes?

Reply:
I think i have to leave this for now... or the import expression will not be understood.